src/olympia/addons/utils.py

```diff
@@ -30,16 +28,7 @@ def _check(name):
         name = normalize_string(name, strip_punctuation=True).lower()
 
         for symbol in amo.MOZILLA_TRADEMARK_SYMBOLS:
-            if waffle.switch_is_active('content-optimization'):
```

Comment:
Should we have a migration to delete this waffle switch?

Reply:
No, there are still other features gated by that switch.